api/config/default.json

```diff
@@ -5,7 +5,7 @@
         "port": 6379
     },
     "spreadsheet": {
-        "id": "1iX0ulEYbyYOy7Dehy3jGmSsWK4JAJA47Hvaa2cF7u1U",
+        "id": "1hQKMblRa0tgFjxN_sTVzaDqmWOk9ZSFsyh473DjAYq0",
```

Comment:
The SID will no longer matter when we begin using structured input, but I inserted the FA24 HAID SID anyway.